src/mame/snk/hng64_v.cpp

```diff
@@ -882,9 +891,12 @@ uint32_t hng64_state::screen_update_hng64(screen_device &screen, bitmap_rgb32 &b
 	}
 #endif
 
-
 	// Initialize some buffers
-	bitmap.fill(m_tcram[0x50/4] & 0x10000 ? m_palette->black_pen() : m_palette->pen(0), cliprect); //FIXME: Is the register correct? check with HW tests
+	if(m_fbcontrol[0] & 0x01) //FIXME: Is the register correct? check with HW tests
+		bitmap.fill(m_palette->pen(0), cliprect); 
```

Comment:
Pretty likely it isn't, if I have to follow the SNK/ADK design then pen(0) must have some special meaning in mixing, cfr. https://wiki.neogeodev.org/index.php?title=Palettes
> The first color of the palette bank ($400000) is the [Reference color](https://wiki.neogeodev.org/index.php?title=Reference_color&action=edit&redlink=1) for the video output. It always has to be pure black ($8000) otherwise monitors won't be happy and other colors won't be displayed correctly.

Reply:
well new register is just a test, it seems correct for the cases I know about.

previous register is actually related to the 3d buffers, and was causing migraine induing flicker in places as it was changing bg colour between black and another value every frame in the samsho games